products/rabbitmq.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: RabbitMQ
+title: VMware RabbitMQ
```

Comment:
-1

> VMware offers a [range of commercial offerings for RabbitMQ](https://tanzu.vmware.com/rabbitmq). This includes a distribution called [VMware RabbitMQ](https://network.pivotal.io/products/p-rabbitmq-for-kubernetes/info) that deploys on Kubernetes or your container Runtime, as well as a version that deploys in [VMware Tanzu Application Service](https://tanzu.vmware.com/tanzu).

VMWare RabbitMQ is something else.

Reply:
i know it i am not the only one :D 
https://github.com/endoflife-date/endoflife.date/pull/2036#issuecomment-1350603622

Reply:
actually it is same with commercial license [ just like oracle java ]